.gitignore

```diff
@@ -71,3 +71,6 @@ crashlytics-build.properties
 # Temporary auto-generated Android Assets
 /[Aa]ssets/[Ss]treamingAssets/aa.meta
 /[Aa]ssets/[Ss]treamingAssets/aa/*
+
+.sln
+.slnx
```

Comment:
1. `.sln` is already ignored (`.slnx` is not)
2. Can someone explain why we would ignore it? I know Unity will just regenerate it but isn't this a file that probably shouldn't be ignored?